challenge_receptionist/src/challenge_receptionist/find_empty_seat.py

```diff
@@ -19,8 +29,11 @@ def __init__(self, robot, seat_ids, room, name=None):
 
         self.room = room
         self.seat_ids = seat_ids
+        self.debug = debug
 
     def _resolve(self):
+        if self.debug:
+            import ipdb;ipdb.set_trace()
```

Comment:
Do you want to keep the debug stuff here? or did you forget to delete it?

Reply:
I wantto keep it, it's optional and off by default